### DISCUSSION TITLE
Update pydantic deploy instructions

### DISCUSSION THREAD

**Discussion by ViniGodoy** [fastapi/fastapi#11879] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
pydantic>=1.2.0,<2.0.0
```

### Description

Since pydantic is now compliant with pydantic 2, shouldn’t these instruction be updated?
https://fastapi.tiangolo.com/deployment/versions/

They still say:
_You can pin Pydantic to any version above 1.0.0 that works for you and below 2.0.0._

### Operating System

Linux, Windows, macOS, Other

### FastAPI Version

Latest

### Pydantic Version

Latest

### Python Version

Any

**Comment by YuriiMotov** [comment]
Done
